src/common/utils/NebulaKeyUtils.h

```diff
@@ -251,6 +258,108 @@ class NebulaKeyUtils final {
         return rawKey.subpiece(0, rawKey.size() - sizeof(int64_t));
     }
 
+    // Only int and double are supported
+    static std::string boundVariant(nebula::cpp2::SupportedType type,
+                                    NebulaBoundValueType op,
+                                    const VariantType& v = 0L) {
+        switch (op) {
+            case NebulaBoundValueType::kMax : {
+                std::vector<unsigned char> bytes = {0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF};
+                return std::string(bytes.begin(), bytes.end());
+            }
+            case NebulaBoundValueType::kMin : {
+                std::string str;
+                str.reserve(sizeof(int64_t));
+                str.append(sizeof(int64_t), '\0');
+                return str;
+            }
+            case NebulaBoundValueType::kSubtraction : {
```

Comment:
For integer,  what's the meaning about subtraction?

Reply:
> If so, why not use the bytes encoded by (int - 1) directly?

Yes, we can do that if just have int type. this method also has double. so using bytes to do it at here.